rskj-core/src/main/java/co/rsk/rpc/netty/Web3HttpStatusCodeProvider.java

```diff
@@ -0,0 +1,48 @@
+package co.rsk.rpc.netty;
+
+import com.googlecode.jsonrpc4j.HttpStatusCodeProvider;
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static com.googlecode.jsonrpc4j.ErrorResolver.JsonError;
+import static com.googlecode.jsonrpc4j.ErrorResolver.JsonError.*;
+
+public class Web3HttpStatusCodeProvider implements HttpStatusCodeProvider {
```

Comment:
Based on [this class](https://github.com/briandilley/jsonrpc4j/blob/master/src/main/java/com/googlecode/jsonrpc4j/DefaultHttpStatusCodeProvider.java) (which we used prior to this change).
FYI: version 1.6 of jsonrpc4j doesn't adjust error handling to the JSON-RPC guidelines, that's why I decided not to update it this PR.